src/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Greeting from './components/Greeting';
+import './App.css';
+
+function App() {
+  return (
+    <>
+      <Greeting />
+    </>
+  );
+}
+
+export default App;
```

Comment:
- [ ] @Obimbo07 Great work adding the greeting app in your app component, however, you did it wrongly, the App component should have a route to access the greeting component like the example below setting the route for Greeting component as '/'
 ` return (
    <Routes>
      <Route path="/" element={<Greeting />} />
    </Routes>
  );`
![image](https://github.com/Obimbo07/hello-rails-frontend/assets/61408860/61552bd2-d520-4b8e-91ff-4258dffae69d)